contract_comm/blockchain_parameters/blockchain_parameters.go

```diff
@@ -113,15 +113,18 @@ func init() {
 func GetMinimumVersion(header *types.Header, state vm.StateDB) (*params.VersionInfo, error) {
 	version := [3]*big.Int{big.NewInt(0), big.NewInt(0), big.NewInt(0)}
 	var err error
-	_, err = contract_comm.MakeStaticCall(
+	comm, err := caller.NewCaller(header, state)
```

Comment:
Just from a naming perspective, this is a bit odd to me. Why call the package `caller` and the `Caller` object `comm` instead of the other way around?

Reply:
It probably makes sense to switch it. It's like that as a hold over from a couple earlier changes.